jekyll/_cci2/insights-glossary.md

```diff
@@ -14,10 +14,6 @@ contentTags:
 This document provides definitions for all the metrics in CircleCI Insights. You
 can review metrics in the Insights dashboard as well as the Insights API.
 
-* TOC
```

Comment:
I noticed there's a note in bold that should be a banner at line 72 (GH won't let me comment there for some reason)